packages/analytics-remote-config/src/types.ts

```diff
@@ -0,0 +1,30 @@
+export interface SessionReplaySamplingConfig {
+  sample_rate: number;
+  capture_enabled: boolean;
+}
+
+export interface SessionReplayRemoteConfig {
+  sr_sampling_config: SessionReplaySamplingConfig;
+}
+
+export type RemoteConfig = SessionReplayRemoteConfig;
+
+export enum ConfigNamespace {
+  SESSION_REPLAY = 'sessionReplay',
+}
+
+export interface RemoteConfigAPIResponse {
```

Comment:
@izaaz @justin-fiedler Also if we make this package fully generic, where does a type like this live? This type is tightly coupled to the response from the Remote Config Service, so to me it makes the most sense to expose it through this package

Reply:
If this is super generic, I'd imagine the type lives closer to the code that cares about this type.